include/crow/middlewares/cors.h

```diff
@@ -155,10 +160,10 @@ namespace crow
         void before_handle(crow::request& /*req*/, crow::response& /*res*/, context& /*ctx*/)
         {}
 
-        void after_handle(crow::request& req, crow::response& res, context& /*ctx*/)
+        void after_handle(request& req, response& res, context& /*ctx*/)
```

Comment:
to be consistent we should always set request& and response& - in before_handle(...) crow::response& and crow::request are used.

Reply:
Reverted `crow::` removal if that's what you're talking about